packages/application-shell/src/utils/location/location.ts

```diff
@@ -21,7 +21,7 @@
  *    mocking system of Jest and no other trickery.
  * */
 const replace = (url: string): void => window.location.replace(url);
-const reload = window.location.reload;
+const reload = (): void => window.location.reload();
```

Comment:
```suggestion
const reload = (...args): void => window.location.reload(...args);
```

Reply:
Accordin to MDN this method doesn't accept args. Do you have a reference stating that it does. Maybe it's just my search skillz.

Reply:
<img width="743" alt="image" src="https://user-images.githubusercontent.com/1110551/77324884-b796a500-6d17-11ea-9de8-40d472bca542.png">


https://developer.mozilla.org/en-US/docs/Web/API/Window/location

Reply:
Ok. Then maybe let's add it to not introduce another change with the fix. Would at least be my way of dealing with it :)